addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -34,16 +34,21 @@
                 <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 col-lg-5 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 80px; --grid-item-padding-x: 40px; z-index: 4;">
                     <div class="s_chart" data-type="doughnut" data-legend-position="none" data-tooltip-display="false" data-stacked="false" data-border-width="2" data-data='{"labels":["First","Second"],"datasets":[{"label":"One","data":["25","75"],"backgroundColor":["o-color-3","o-color-2"],"borderColor":["",""]}]}' data-snippet="s_chart" data-name="Chart">
                         <canvas style="box-sizing: border-box;" width="456" height="228"/>
+                        <p><br/></p>
+                        <h4 class="display-1-fs" style="text-align: center;">75%</h4>
+                        <p class="o_small text-600" style="text-align: center;">75% of clients use the service for over a decade consistently.<br/>This showcases remarkable loyalty and satisfaction with the quality provided.</p>
```

Comment:
- This sentence does not seem to use proper english to me, should it not be "have been using" or something like that?
- is it something possible to achive in the editor? :thinking: Adding text *inside* the s_chart snippet? If it is possible, I am not sure it should be... :grimacing: